cmd/dockerd/daemon.go

```diff
@@ -512,6 +524,23 @@ func (cli *daemonCLI) stop() {
 	})
 }
 
+// sanitizedConfig sanitizes the config for printing, and returns it as
+// a JSON-encoded string.
+//
+// TODO(thaJeztah): move redacting the config to a utility in the config package.
+func sanitizedConfig(cfg config.Config) (string, error) {
```

Comment:
Actually, reading my TODO; perhaps it would make sense to get this done already? I can have a quick peek